packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -6,7 +6,8 @@ import 'dart:async';
 import 'dart:convert' show json;
 import 'dart:developer' as developer;
 import 'dart:io' show exit;
-import 'dart:ui' show saveCompilationTrace;
+// Before adding any more dart:ui imports, pleaes read the README.
```

Comment:
Do we need to update that README to explain why the new dependency on Window is ok?

Reply:
I'll leave that README modification to @Hixie